src/logtab.cpp

```diff
@@ -39,18 +39,20 @@ void LogTab::InitTreeView(const EventListPtr events)
 
     m_bar->ShowMessage(QString("%1 events loaded").arg(QString::number(m_treeModel->rowCount())), 3000);
 
+    bool isPlainText = (events->size() > 0 && events->at(0)["k"].toString().isEmpty());
```

Comment:
The other way is to go through the tree model, which would require getting the model index first and less efficient.